### ISSUE TITLE
V4 intellisense not working in a monorepo v0.12.9 and after

### ISSUE THREAD

**Issue by sibbng** [tailwindlabs/tailwindcss-intellisense#1054]
**What version of VS Code are you using?**

v.1.93.1

**What version of Tailwind CSS IntelliSense are you using?**

0.12.9
Tried 0.12.10 and pre-release too

**What version of Tailwind CSS are you using?**

4.0.0-alpha.24

**What package manager are you using?**

pnpm

**What operating system are you using?**

Windows

**Tailwind config**

```js
/** @type {import('tailwindcss').Config} */
export default {
  content: ["./client/src/**/*.{js,jsx,ts,tsx}"],
  theme: {
    extend: {},
  },
  plugins: [],
};

```

**VS Code settings**

```json
  "tailwindCSS.experimental.classRegex": [
    ["clsx\\(([^)]*)\\)", "(?:'|\"|`)([^']*)(?:'|\"|`)"]
  ],
```

**Reproduction URL**

I thought the issue is obvious, if it's not I would happily provide a repro.

**Describe your issue**

Hi, I'm using v4 in a monorepo and I have 2 issues:

1- Intellisense not working v0.12.9 and after (including pre-release)
2- v0.12.8 (working version) only working if I have `@config "path/to/tailwind.conf.ts"` in my css file. I thought config file is not mandatory in v4.

### Workaround
- Moving the entry css file to root directory fixes intellisense, Related comment https://github.com/tailwindlabs/tailwindcss-intellisense/issues/1033#issuecomment-2362876115

**Comment by philipp-spiess** [comment]
Hey @sibbng! Do you mind checking if the logs has any information on this? You can access it via View > Output and then picking the "Tailwind CSS IntelliSense" extension in the dropdown of the output module.

**Comment by brandongit2** [comment]
Yep, I can confirm this same issue is affecting us too. This feels like a pretty significant bug. This being outside the main Tailwind repo, it would be great to confirm that the Tailwind team has eyes on this!

**Comment by thecrypticace** [comment]
@sibbng Is this still a problem for you on the latest version of the extension?

**Comment by thecrypticace** [comment]
Ah excellent! Gonna close this one then 👍

**Comment by Avinash-Tallapaneni** [comment]
@sibbng @thecrypticace Hi, The issue still exists, im using monorepo setup from Shadcn with Nextjs app
